### PR TITLE
Restore tests for old application-gateway as a new project

### PR DESCRIPTION
**Pull request by koala7659**
Changes:

- [x] Restore old way of testing of legacy application gateway as new test project application-gateway-legacy-tests

- [x] Remove verification of Istio Resources during testing  

- [x] Update charts to use the image from build for application-gateway-legacy-tests

See [the issue](https://github.com/kyma-project/kyma/issues/9094)

### CODE REVIEW

File: tests/application-gateway-legacy-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Application Gateway Tests
+
+## Overview
+
+This project contains the acceptance tests for the Kyma Application Gateway.
```
Comment (majakurcius) [inline tests/application-gateway-legacy-tests/README.md:5]:
```suggestion
This project contains the acceptance tests for the Kyma Application Gateway in the legacy mode.
```

File: tests/application-gateway-legacy-tests/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Prerequisites
+
+The project requires Go 1.8 or higher.
+
+## Usage
+
+Environment parameters used by the tests:
+
+| Name | Required | Default | Description | Possible values |
```
Comment (majakurcius) [inline tests/application-gateway-legacy-tests/README.md:15]:
```suggestion
| Name | Required | Default | Description | Example value |
```

Comment (majakurcius) [inline tests/application-gateway-legacy-tests/README.md:15]: See the [guidelines for tables](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md#tables).
Although we have some inconsistencies in our documentation, our tables generally follow the order: 
```
| Parameter      | Required | Description   | Default value |       {PROFILE} value       | Example value   |
|----------------|:--------:|---------------|---------------|-----------------------------|-----------------|
| {Param's name} | {Yes/No} | {Description} |      None     | {Value for the given prof.} | {Example value} |
```
For consistency's sake, we should stick to the same order when creating tables.

File: tests/application-gateway-legacy-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+|------|----------|---------|-------------|-----------------|
+| **APPLICATION** | Yes | None | The name of the Application to test | `my-application` |
+| **NAMESPACE** | Yes | None | The Namespace in which the test Application will operate | `kyma-integration` |
+| **MOCK_SERVICE_PORT** | Yes | None |  Port number used by the mock service created by the test | `8080` |
+| **TEST_EXECUTOR_IMAGE** | No | Same version as the Helm test image  | Image name of the test executor created by the Helm test  | `user/my-image:1.0.0` |
```
Comment (majakurcius) [inline tests/application-gateway-legacy-tests/README.md:20]:
```suggestion
| **TEST_EXECUTOR_IMAGE** | No | Version matching the Helm test image  | Image name of the test executor created by the Helm test  | `user/my-image:1.0.0` |
```

File: tests/application-gateway-legacy-tests/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## Usage
+
+Environment parameters used by the tests:
+
+| Name | Required | Default | Description | Possible values |
+|------|----------|---------|-------------|-----------------|
+| **APPLICATION** | Yes | None | The name of the Application to test | `my-application` |
+| **NAMESPACE** | Yes | None | The Namespace in which the test Application will operate | `kyma-integration` |
+| **MOCK_SERVICE_PORT** | Yes | None |  Port number used by the mock service created by the test | `8080` |
```
Comment (majakurcius) [inline tests/application-gateway-legacy-tests/README.md:19]:
```suggestion
| **MOCK_SERVICE_PORT** | Yes | None | Number of the port used by the mock service created by the test | `8080` |
```

File: tests/application-gateway-legacy-tests/README.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+The project requires Go 1.8 or higher.
+
+## Usage
+
+Environment parameters used by the tests:
+
+| Name | Required | Default | Description | Possible values |
+|------|----------|---------|-------------|-----------------|
+| **APPLICATION** | Yes | None | The name of the Application to test | `my-application` |
+| **NAMESPACE** | Yes | None | The Namespace in which the test Application will operate | `kyma-integration` |
```
Comment (majakurcius) [inline tests/application-gateway-legacy-tests/README.md:18]:
```suggestion
| **NAMESPACE** | Yes | None | Namespace in which the test Application will operate | `kyma-integration` |
```
We tend to skip the definite article at the beginning of the description in such columns for improved readability.

File: tests/application-gateway-legacy-tests/README.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+The project requires Go 1.8 or higher.
+
+## Usage
+
+Environment parameters used by the tests:
+
+| Name | Required | Default | Description | Possible values |
+|------|----------|---------|-------------|-----------------|
+| **APPLICATION** | Yes | None | The name of the Application to test | `my-application` |
```
Comment (majakurcius) [inline tests/application-gateway-legacy-tests/README.md:17]:
```suggestion
| **APPLICATION** | Yes | None | Name of the Application to test | `my-application` |
```

File: tests/application-gateway-legacy-tests/README.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+| **NAMESPACE** | Yes | None | The Namespace in which the test Application will operate | `kyma-integration` |
+| **MOCK_SERVICE_PORT** | Yes | None |  Port number used by the mock service created by the test | `8080` |
+| **TEST_EXECUTOR_IMAGE** | No | Same version as the Helm test image  | Image name of the test executor created by the Helm test  | `user/my-image:1.0.0` |
+
```
Comment (majakurcius) [inline tests/application-gateway-legacy-tests/README.md:21]:
```suggestion
```

**Review by majakurcius — COMMENTED** [review]
A few minor comments/suggestions 
image

File: tests/application-gateway-legacy-tests/README.md (hunk 2 above)
Comment (koala7659) [reply tests/application-gateway-legacy-tests/README.md]: At first I will switch Default and Description columns. I will lookup for PROFILE values then.

File: tests/application-gateway-legacy-tests/README.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+## Usage
+
+Environment parameters used by the tests:
+
+| Name | Required | Description | Default | Example value |
+|------|----------|---------|-------------|-----------------|
+| **APPLICATION** | Yes | Name of the Application to test | None | `my-application` |
+| **NAMESPACE** | Yes | Namespace in which the test Application will operate | None | `kyma-integration` |
+| **MOCK_SERVICE_PORT** | Yes | Number of the port used by the mock service created by the test | None | `8080` |
+| **TEST_EXECUTOR_IMAGE** | No | Image name of the test executor created by the Helm test | Version matching the Helm test image | `user/my-image:1.0.0` |
```
Comment (majakurcius) [inline tests/application-gateway-legacy-tests/README.md:20]:
```suggestion
| **TEST_EXECUTOR_IMAGE** | No | Name of the test executor image created by the Helm test | Version matching the Helm test image | `user/my-image:1.0.0` |
```

File: tests/application-gateway-legacy-tests/README.md (hunk 2 above)
Comment (majakurcius) [reply tests/application-gateway-legacy-tests/README.md]: You don't need to apply all the columns. Choose the ones you need, just stick to the order. The profile value refers to tables like [this one](https://sap.sharepoint.com/sites/100093/_layouts/15/Doc.aspx?sourcedoc={d9b81bfd-650d-4e42-9f62-43220886f783}&action=edit&wd=target%28SIG%20Content.one%7C0a2f8356-73ae-4f4a-adf6-0a3617f99e51%2FRelease%20notes%201.16%20%28input%5C%29%7C22d7a1e1-38b5-449d-8b8d-b42c3745632c%2F%29).

**Review by majakurcius — DISMISSED** [review]
🚀

**Review by rafalpotempa — CHANGES_REQUESTED** [review]

**Comment by rafalpotempa** [comment]
I worry that the upgrade may require some adjustments in code. :/

The rest is ok ✅

File: tests/application-gateway-legacy-tests/Gopkg.toml [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.15.2"
+
+[[override]]
+  name = "k8s.io/api"
+  version = "kubernetes-1.15.2"
+
+[[override]]
+  name = "k8s.io/apimachinery"
+  version = "kubernetes-1.15.2"
```
[the remark this answers is not in the document]
Comment (koala7659) [reply tests/application-gateway-legacy-tests/Gopkg.toml]: Updated to 1.15.12

**Review by majakurcius — APPROVED** [review]

**Review by rafalpotempa — APPROVED** [review]

**Review by pbochynski — APPROVED** [review]

**Pull request merged**